refactor/common.py

```diff
@@ -176,6 +176,16 @@ def find_indent(source: str) -> tuple[str, str]:
     return source[:index], source[index:]
 
 
+def find_common_chars(source: str, compare: str) -> str:
+    """Finds the common characters starting the 2 strings"""
+    index: int = 0
+    for index, char in enumerate(source):
+        if index > len(compare) or char != compare[index]:
+            index -= 1
+            break
+    return source[:index+1]
```

Comment:
This is not common characters but rather the common prefix right? In that case, we should probably refactor (and add some tests) OR use the standard library version ([`os.path.commonprefix`](https://docs.python.org/3/library/os.path.html#os.path.commonprefix)).

Reply:
Great, let's use `os.path.commonprefix`